docs/graphql-admin/locales.mdx

```diff
@@ -0,0 +1,256 @@
+# GraphQL Admin: Locale overview
+
+## Introduction
+
+The BigCommerce GraphQL Admin API provides the same access to the locales field as the REST Management [Store Locale](/docs/rest-management/settings/store-locale) API. Having access to locale data in the graph alongside related data makes it easier to build headless storefronts.
+
+## Getting started
+
+The GraphQL Admin API's allows merchants to add and remove locales per each sales channel. 
```

Comment:
you can also update locales via this API.
change status and whether locale is default or not